common/djangoapps/student/views/management.py

```diff
@@ -195,6 +196,13 @@ def index(request, extra_context=None, user=AnonymousUser()):
     # Add marketable programs to the context.
     context['programs_list'] = get_programs_with_type(request.site, include_hidden=False)
 
+    journal_info = {}
+    journal_info['journals'] = get_journals(request.site)
+    journal_info['journals_root_url'] = get_journals_root_url()
+    journal_info['journal_bundles'] = get_journal_bundles(request.site)
```

Comment:
These 4 lines are duplicated again below.  Can we create a helper for this within `openedx.features.journals.api`?

Reply:
fixed